### DISCUSSION TITLE
Welcome to EDAutopilot Discussions!

### DISCUSSION THREAD

**Discussion by Matrixchung** [Matrixchung/EDAutopilot#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below and tell us about what problem or idea you have.

**Comment by User989811** [comment]
"Hello Matrixchung, unfortunately, I can't get EDAutopilot-v2-master to run. I am currently using Python 3.12 and had to install PySide6 instead of PySide2. When starting, I receive the following message: 

'qt.qpa.window: SetProcessDpiAwarenessContext() failed: Der Vorgang wurde erfolgreich beendet.
Qt's default DPI awareness context is DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2. If you know what you are doing, you can overwrite this default using qt.conf (https://doc.qt.io/qt-6/highdpi.html#configuring-windows).' 

Perhaps you have a tip for me on how to get EDAutopilot-v2 up and running?"

**Reply by Matrixchung** [reply]
Hello User989811,
As far as I know, PySide6 represents the C++ Qt6 while PySide2 represents Qt5. It seems that the whole project was developed based on Qt5, and I'm not sure how much Qt6 changes the API or some other implementation. Maybe the only way to get the project running is switching back to earlier Python version and using PySide2.

**Comment by User989811** [comment]
thanks for the answer! I will try


Matr1x. ***@***.***> schrieb am So. 19. Nov. 2023 um 08:12:

**Comment by User989811** [comment]
Hello Matrixchung,
I wanted to share that I've successfully managed to run EDAutopilot-v2-master using PyQt6.uic's loadUiType. This was achieved with both PyQt6 and PySide6 libraries.
The combination of PyQt6 and PySide6, along with the loadUiType functionality, has allowed me to run EDAutopilot-v2 smoothly.

I've been trying to access the game.py file, but it seems that the provided link is broken or no longer exists. Could someone please let me know where I can find the game.py file? 

Thank you in advance for your assistance!
Best regards,

**Reply by Matrixchung** [reply]
Which game.py do you refer to? As far as I know, the main python file is called gameui.py.

**Comment by alyons19995** [comment]
I am a complete idiot, but how do you get this to work? I downloaded it but nothing is working

**Reply by Matrixchung** [reply]
This software is no longer under active development, so you might encounter issues mainly because of game update, like change in button position/size/color, etc.

**Comment by EpiloxGaming** [comment]
Quick questions from a complete noob here... 

How likely is a ban, will playing in solo help? and how hard would it to be to add the functionality to plot a route and it follow that as well as wait at refuel stars? So for example to get from the bubble to the middle or beyond?

**Reply by Matrixchung** [reply]
I haven't got any warn for over one year of development process, and I do recommend you run the script in solo mode in case of any uncontrolled situation. It might be easy to add function since the script provides all the interfaces you need to communicate with base system.

**Comment by Kilativ12** [comment]
Hello, the issue is that I seem to be an idiot because I can't figure out how to install the script, + I don't understand Python, which complicates the situation. It would be great if there was a video guide for people like me.
